### ISSUE TITLE
Variable Wemb/Adam/ does not exist, or was not created with tf.get_variable(). Did you mean to set reuse=None in VarScope?

### ISSUE THREAD

**Issue by chaiyixuan** [brianhuang1019/RL-Chatbot#7]
code：
with tf.variable_scope(tf.get_variable_scope(), reuse=None):
            train_op = tf.train.AdamOptimizer(self.lr).minimize(loss)

error：
Variable Wemb/Adam/ does not exist, or was not created with tf.get_variable(). Did you mean to set reuse=None in VarScope?

I dont kown why

**Comment by brianhuang1019** [comment]
你是用 train 好的 model 跑的嗎？
還有 tensorflow 版本是否正確

**Comment by chaiyixuan** [comment]
My tensorflow is 1.4, I change my code and it works!

def build_model(self):
        with tf.variable_scope(tf.get_variable_scope()):# add this line in model.py
            word_vectors = tf.placeholder(tf.float32, [self.batch_size, self.n_encode_lstm_step, self.dim_wordvec])

**Comment by brianhuang1019** [comment]
Good Job!

**Comment by estelleaf** [comment]
Hi ,

Can you be more specific chaiyixuan ? It still does not work for me...

Thanks
